flyteadmin/pkg/rpc/adminservice/middleware/recovery_interceptor_test.go

```diff
@@ -0,0 +1,38 @@
+package middleware
```

Comment:
I'm open to changing where this lives but it feels like there should be a middleware package, and any interceptors should move here imo.

Reply:
Sounds good!